lookout/style/format/rules.py

```diff
@@ -153,16 +153,16 @@ class TrainableRules(BaseEstimator, ClassifierMixin):
 
     _log = logging.getLogger("TrainableRules")
 
-    def __init__(self,
-                 base_model: Union[DecisionTreeClassifier, RandomForestClassifier],
-                 prune_branches=True, prune_branches_algorithm="top-down-greedy",
+    def __init__(self, base_model_name: str, prune_branches=True,
```

Comment:
please rename to `base_model_type` and annotate as `Union[Type[DecisionTreeClassifier], Type[RandomForestClassifier]]` so that we pass in the type

Reply:
there can be problems during the serialization, let's address them later